fetch-api/fetch-referrer-policy.html

```diff
@@ -0,0 +1,25 @@
+---
+feature_name: Fetch Referrer Policy
+chrome_version: 52
+feature_id: 5692236742262784
+---
+
+<h3>Background</h3>
+<p>
+  This sample shows how requests made via the  Fetch API can use a specific
```

Comment:
It's non-printing, intentional two-space indentation, to set the text content apart from the `<p>` parent.